provider/gce/instance_information.go

```diff
@@ -0,0 +1,64 @@
+// Copyright 2016 Canonical Ltd.
+// Licensed under the AGPLv3, see LICENCE file for details.
+
+package gce
+
+import (
+	"strconv"
+
+	"github.com/juju/errors"
+	"github.com/juju/utils/arch"
+
+	"github.com/juju/juju/constraints"
+	"github.com/juju/juju/environs"
+	"github.com/juju/juju/environs/instances"
+)
+
+var _ environs.InstanceTypesFetcher = (*environ)(nil)
+var virtType = "kvm"
```

Comment:
if I use a const I cannot pass its addres afterwards (instances.InstanceType.VirtType is a pointer)